tests/mount.spec.ts

```diff
@@ -27,7 +27,16 @@ describe('mount: general tests', () => {
 
     expect(wrapper.html()).toBe('<div>hello</div>')
   })
+  it('if an error is thrown during mounting, the first error is thrown.', () => {
```

Comment:
nit: can you name the test with the usual convention `should...`? For example `should throw the first error encountered when mounting the component`.
Can you also add an empty line above and below the test as we usually do?